src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,42 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_ARRAY_VALUE = 10;
+    private int size = 0;
+    private final Object[] keyArray;
+    private final Object[] valueArray;
```

Comment:
```suggestion
    private final Object[] keys;
    private final Object[] values;
```
We'll already know it's an array if the name is just plural)